packages/components/card/src/inline-entry-card/InlineEntryCard.tsx

```diff
@@ -0,0 +1,65 @@
+import React from 'react';
+import { css, cx } from 'emotion';
+import tokens from '@contentful/f36-tokens';
+import { getEntityStatusStyles } from '@contentful/f36-core';
+import { Text } from '@contentful/f36-typography';
+
+import { BaseCard } from '../base-card/BaseCard';
+import type { EntryCardInternalProps } from '../entry-card/EntryCard';
+import { getInlineEntryCardStyles } from './InlineEntryCard.styles';
+import { CardActions } from '../base-card/CardActions';
+
+const generateStyles = ({
+  status,
+}: {
+  status: InlineEntryCardProps['status'];
+}) => {
+  const statusColors = getEntityStatusStyles({ status });
+
+  return css({
+    '&::before': {
+      backgroundColor: statusColors.color,
+      bottom: 0,
+      content: '""',
+      display: 'block',
+      left: 0,
+      position: 'absolute',
+      top: 0,
+      width: tokens.spacing2Xs,
+    },
+  });
+};
```

Comment:
Shouldn't we move all these `getStyles` functions to the `.styles.ts` files?

Reply:
Yeah. Changed to a nice pattern I picked up from @bgutsol 👍